src/waiter-reducer.js

```diff
@@ -112,6 +110,7 @@ const reducerMap = {
     elapsedTime: getTime() - state.startTime,
     lastModified: getTime(),
   }),
+
   [t.CLEAR]: (state) => ({
     ...state,
 
```

Comment:
Was wondering about params here. clearWaiter is about clearing the request lifecycle. The params and requestCreator should be available to use again.